tools/plotDetectors.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+def plot(radius, count, nPackets, dectType):
+    print(radius)
+    print(count)
+    #print(count)        
+    totalCounts = sum(count)
+    print("Detector Type : " + dectType)
+    print("Total Diffuse : " + str(totalCounts/nPackets))
```

Comment:
Use f-strings instead of str(foo) https://realpython.com/python-f-strings/#doing-string-interpolation-with-f-strings-in-python